### PR TITLE
[AAE-8764] Enable left labels in text, number and dropdown cloud widget

### PR DESCRIPTION
**Pull request by rbarroso**
**Please check if the PR fulfills these requirements**

> - [X] The commit message follows our [guidelines](https://github.com/Alfresco/alfresco-ng2-components/wiki/Commit-format)
> - [ ] Tests for the changes have been added (for bug fixes / features)
> - [ ] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (check one with "x")

> - [ ] Bugfix
> - [ ] Feature
> - [ ] Code style update (formatting, local variables)
> - [ ] Refactoring (no functional changes, no api changes)
> - [ ] Build related changes
> - [ ] Documentation
> - [X] Other... Please describe:
Refactor styles

**What is the current behaviour?** (You can also link to an open issue here)

**What is the new behaviour?**
Text, number and dropdown widget accept form compact mode showing their labels in the left side of the input

**Does this PR introduce a breaking change?** (check one with "x")

> - [ ] Yes
> - [x] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:

### CODE REVIEW

File: lib/core/form/components/widgets/core/form-field.model.ts [hunk 1]
```diff
@@ -76,6 +76,7 @@ export class FormFieldModel extends FormWidgetModel {
     rule?: FormFieldRule;
     selectLoggedUser: boolean;
     groupsRestriction: string[];
+    compactForm: boolean = false;
```
Comment (eromano) [inline lib/core/form/components/widgets/core/form-field.model.ts:79]: Better to not use compact as a definition for this functionality. The compact is actually another thing related to form density present in another functionality https://alfresco.atlassian.net/browse/AAE-6939

**Review by eromano — CHANGES_REQUESTED** [review]
missing unit test plus a comment above

File: lib/core/form/components/form-renderer.component.scss [hunk 2]
```diff
@@ -106,6 +106,34 @@
             font-weight: bold;
         }
     }
+
+    &-compact-field-container {
+        display: flex;
+        font-size: 12px;
```
Comment (pmartinezga) [inline lib/core/form/components/form-renderer.component.scss:112]: shouldn't we use font size relative to theme variables?
```suggestion
        font-size: var(--theme-caption-font-size);
```

File: lib/core/form/components/form-renderer.component.scss [hunk 3]
```diff
@@ -106,6 +106,34 @@
...
+        padding-bottom: 0;
+
+        div:nth-child(2) {
+            flex: 1;
+
+            mat-form-field {
+                font-size: 12px;
```
Comment (pmartinezga) [inline lib/core/form/components/form-renderer.component.scss:119]: same here

**Review by pmartinezga — CHANGES_REQUESTED** [review]

File: lib/core/form/components/form-renderer.component.scss (hunk 2 above)
Comment (rbarroso) [reply lib/core/form/components/form-renderer.component.scss]: Not need anymore

File: lib/core/form/components/form-renderer.component.scss (hunk 3 above)
Comment (rbarroso) [reply lib/core/form/components/form-renderer.component.scss]: Not need anymore

**Review by pmartinezga — APPROVED** [review]

**Review by nazarethjim — APPROVED** [review]

**Review by eromano — APPROVED** [review]

**Pull request merged**